docker/production/docker-compose.yml

```diff
@@ -20,12 +20,14 @@ services:
     volumes:
       - /etc/localtime:/etc/localtime:ro
       ## Keep configs here.
-      - ./config:/root/.stash
+      - /your/path/to/Stash/config:/root/.stash
```

Comment:
maybe  
/path/to/your/stash/config  
instead? same for the rest

Reply:
I would prefer the relative path personally. This way we don’t inadvertently start filling people’s host file system with junk.